### DISCUSSION TITLE
After update to 2.9.1.366 I'm unable to add tumblr blogs

### DISCUSSION THREAD

**Discussion by complaintspleasetakeanumber** [TumblThreeApp/TumblThree#391] [Q&A (Question & Answer)]

**Comment by thomas694** [comment]
Can you provide us more details?
Are you using the same format as the URLs of your existing blogs?

**Comment by complaintspleasetakeanumber** [comment]
Thanks for the prompt response. I believe I’ve figured it out.

I previously add blogs using this format: https://mannis-new-blog.tumblr.com/likes

After the update it inly accepts this format: https://mannis-new-blog.tumblr.com

Also, I used to be able to copy to the clipboard and it would add blog to TumbrThree. Not happening now.

Thanks,
Cornell

**Comment by thomas694** [comment]
The former URL was accepted in the past... Maybe we should make it possible again...

When using the new-style URLs, the copy & paste and the clipboard monitor still work.

**Comment by thomas694** [comment]
The URL format check should be less strict now.
But this URL format still doesn't work, because it points to the user's likes which can only be downloaded when using the format:
https://www.tumblr.com/liked/by/mannis-new-blog
In the past the user's blog was (wrongly) added instead.
